### PR TITLE
Fix typos according to reviewdog report.

### PR DESCRIPTION
**Pull request by mindest**
### Description
Fix typos based on reviewdog report but with some exceptions/corrections.

### CODE REVIEW

**Review by snnn — DISMISSED** [review]

File: cmake/patches/composable_kernel/Fix_Clang_Build.patch [hunk 1]
```diff
@@ -44,7 +44,7 @@ index c23746e7f..bc326c8b5 100644
  find_package(HIP REQUIRED)
  # Override HIP version in config.h, if necessary.
 @@ -269,12 +248,6 @@ if( DEFINED CK_OVERRIDE_HIP_VERSION_PATCH )
-     message(STATUS "CK_HIP_VERSION_PATCH overriden with ${CK_OVERRIDE_HIP_VERSION_PATCH}")
+     message(STATUS "CK_HIP_VERSION_PATCH overridden with ${CK_OVERRIDE_HIP_VERSION_PATCH}")
  endif()
  message(STATUS "Build with HIP ${HIP_VERSION}")
 -link_libraries(hip::device)
```
Comment (pengwa) [inline cmake/patches/composable_kernel/Fix_Clang_Build.patch:50]: ?

Comment (mindest) [reply cmake/patches/composable_kernel/Fix_Clang_Build.patch:50]: It is original diff of the patch file, no change in this line.

**Review by pengwa — APPROVED** [review]

**Review by snnn — APPROVED** [review]

**Comment by mindest** [comment]
@microsoft/onnxruntime-mlas could you help take a look? Thanks.

**Review by fajin-corp — APPROVED** [review]
:shipit:

**Pull request merged**